src/routes/pages/en/setup-zapping-wallet.md

```diff
@@ -0,0 +1,66 @@
+## [§](#understanding-keys) Understanding keys
+Using Bitcoin's Lightning Network, you can send tiny fractions of a bitcoin, known as satoshis (or sats), around the world, at the speed of light, with near-zero fees. On Nostr, these transactions are called zaps.
+
+Even if you're new to Nostr, you've probably already seen at least one mention of zaps. You can zap someone if they make you laugh, or they give you good advice, or if they post something that adds value to your experience. You can even zap someone for no reason at all! And the reverse is also true: if you contribute interesting content to the nostr community, it won't be long before strangers on the internet (read: new friends) are sending you bitcoin.
+
+## [§](#custodial-vs-self) Custodial vs Self-Custodial Wallets
+
+The first thing you'll need is a bitcoin lightning wallet. This is the wallet where your zap transactions come from. There are many options, but one of the most important distinctions to understand is whether the wallet is custodial or self-custodial. Each comes with trade-offs, but we typically recommend that brand-new users first set up a custodial wallet to get comfortable with using zaps and managing their funds.
+
+As you get more comfortable (or once the funds in your custodial wallet reach a level that you want to take full control of) you can set up a self-custodial wallet.
+
+### [§](#custodial) Custodial Wallets
+A custodial wallet is one where the wallet developer holds the keys to the bitcoin, and your account is essentially authorized to send and receive that bitcoin – whether to another user or to a different wallet that you fully control (i.e. a self-custodial wallet). 
+
+Two popular examples are [Wallet of Satoshi](https://www.walletofsatoshi.com/) and [Alby](https://getalby.com). Both wallets provide fast, convenient, and user-friendly zapping wallets. You can download Wallet of Satoshi from your mobile app store, and you can access Alby via the Chrome browser extension store.
+
+[§](#wallet-of-satoshi)**Wallet of Satoshi**
+
+After you download Wallet of Satoshi, open the app and tap "Receive". This will bring up a reusable QR Code, as well as your human-readable lightning receiving address. It will look like an email address (NB: it's not actually an email address so don't try and send email there). For example, this article's author can be zapped at `bostonwine@walletofsatoshi.com`. Copy your address and return to Nostr.
+
+| Main Screen  | Receive Screen |Example client|
+| ------------- | ------------- |------------- |
+| <img src="https://cdn.nostr.build/i/955e1fd028d64941b80ed0b423a07541a2af8f14919c73e0add93511e3620477.jpg" alt="Main Screen" width="200"/>  | <img src="https://cdn.nostr.build/i/2d4f6ffa7a237cf93fca9aff37eca0011ba473b8f3da013f5fda786c93693b87.jpg" alt="Receive Screen" width="200"/>  |<img src="https://cdn.nostr.build/i/34241da3022d061a8159268f05e0cb3c8aa53934a55c5cc8c76effbfc7b625cc.jpg" alt="Example client" width="200"/>
```

Comment:
I think I'd spread these images out. The client image should be down below in the section on Going back to the nostr client. 

The other images for WoS belong here but I think I'd just put the images inline in the page so that they can flow better with different screen sizes. 

You can also use markdown to include images. `![Alt text for the image](url-to-the-image)`

Reply:
WoS images are inline, Damus screenshot is moved down to "return to client", and I've removed the size params.

Thanks for the insight about markdown for images. Is there a technical reason why it's better/worse than html, or is it typically just better to keep it all in one language?

I've updated the image links to use markdown.

I removed the width reduction but the images look BIG in the preview. Maybe they'll look fine on a live site, but let me know what you think. Submitting changes now.